solr/core/src/test/org/apache/solr/search/SolrIndexSearcherTest.java

```diff
@@ -189,12 +206,95 @@ public void testMinExactHitsWithMaxScoreRequested() throws IOException {
       cmd.setMinExactHits(2);
       cmd.setFlags(SolrIndexSearcher.GET_SCORES);
       cmd.setQuery(new TermQuery(new Term("field1_s", "foo")));
-      searcher.search(new QueryResult(), cmd);
       QueryResult qr = new QueryResult();
       searcher.search(qr, cmd);
       assertMatchesGraterThan(NUM_DOCS, qr);
       assertNotEquals(Float.NaN, qr.getDocList().maxScore());
       return null;
     });
   }
+  
+  public void testMinExactWithFilters() throws Exception {
+    
+    h.getCore().withSearcher(searcher -> {
+      //Sanity Check - No Filter
+      QueryCommand cmd = new QueryCommand();
+      cmd.setMinExactHits(1);
+      cmd.setLen(1);
+      cmd.setFlags(SolrIndexSearcher.NO_CHECK_QCACHE | SolrIndexSearcher.NO_SET_QCACHE);
```

Comment:
Curious; why are you writing tests with this low-level way vs testMinExactHitsDisabledByCollapse which I wrote in a more common higher level style that is more succinct?  Is it only for this NO_CHECK_QCACHE distinction?  Can that be done simply by disabling the cache?  Not a big deal but just want to know your point of view.

Reply:
just trying to make the test cover specifically what I was working on. It can definitely be done with a higher level test or integration test.